source/extensions/filters/http/ext_proc/mutation_utils.h

```diff
@@ -14,8 +14,8 @@ namespace ExternalProcessing {
 class MutationUtils : public Logger::Loggable<Logger::Id::filter> {
 public:
   // Convert a header map until a protobuf
-  static void buildHttpHeaders(const Http::HeaderMap& headers_in,
-                               envoy::config::core::v3::HeaderMap& headers_out);
+  static void headersToProto(const Http::HeaderMap& headers_in,
+                             envoy::config::core::v3::HeaderMap& proto_out);
```

Comment:
Old code but I think unless there is a good reason this should be returning the HeaderMap instead of taking an out param?

Reply:
Interesting question, actually. The only places we call this method we're using it to set the fields of an embedded message field in protobuf. So I think that if we change it, then the code that calls this function has to first get a pointer to an embedded message by calling "mutable_whatever" on the field, and then needs to copy the result of this function into the result. Doing it this way saves the copy which seems like a good thing.

Reply:
Alright that sounds like a good enough reason for me :)